testsuite/tests/kuadrant/test_rate_limit_authz.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
I believe this test file should be under the limitador package

Reply:
It is correctly placed under the Kuadrant package. The reason is that it uses both Limitador (RLP) and Authorino (OIDC identity) features.